tools/src/wyvern/target/corewyvernIL/support/VarGenContext.java

```diff
@@ -30,8 +40,16 @@ public String endToString() {
 	public Expression lookupExp(String varName) {
 		if (varName.equals(var))
 			return expr;
-		else
+		/*
+		else if (genContext.getType(varName) != null) {
+			String objName = genContext.getType(varName);
+			return new FieldGet(new Variable(objName), varName);
+		}  
+		*/
+		// You are right, we do not need this.
```

Comment:
Ok--better just to remove it rather than comment it out. But thanks for your reply!